extensions/extensions-docker/aissemble-hive/aissemble-hive-service/pom.xml

```diff
@@ -15,7 +15,7 @@
     <packaging>docker-build</packaging>
 
     <properties>
```

Comment:
I: this was ultimately resolving as `target/dockerbuild/jars//*` - so I cleaned it up